disco_aws_automation/disco_elb.py

```diff
@@ -198,21 +198,38 @@ def get_or_create_elb(self, hostclass, security_groups, subnets, hosted_zone_nam
         elb_name = DiscoELB.get_elb_name(self.vpc.environment_name, hostclass, testing=testing)
         elb = self.get_elb(hostclass, testing=testing)
 
-        elb_protocols = str(elb_protocols).split(',')
-        elb_ports = str(elb_ports).split(',')
-
         if not elb:
             logger.info("Creating ELB %s", elb_name)
 
+            if len(instance_protocols) != len(instance_ports):
+                raise CommandError(
```

Comment:
This is documented as "a problem with an SSH command".  I think any old subclass of EasyExit would suffice for this, AsiaqConfigError being a sort of obvious fit (albeit with a description that indicates that it's restricted to missing files—that's not a terrible thing to change, though).